src/site/content/en/blog/preconnect-and-dns-prefetch/index.md

```diff
@@ -0,0 +1,142 @@
+{% endAside %}
+
+## Use-cases
+
+### Knowing *where from*, but not *what* you're fetching
+
+Due to versioned dependencies, you sometimes end up in a situation where you know you'll be requesting a resource from a particular CDN, but not the exact path for it. 
+
+<figure class="w-figure w-figure--center">
+<img src="versioned-url.png" style="max-width:450px" alt="A url of a script with the version name.">
+<figcaption>An example of a versioned URL</figcaption>
+</figure>
+
+The other common case is loading images from an [image CDN](/image-cdns), where the exact path for an image depends on media queries or runtime feature checks on the user's browser.
+
+<figure class="w-figure w-figure--center">
+<img src="image-cdn-url.png" alt="An image CDN URL with the parameters size=300x400 and quality=auto.">
+<figcaption>An example of an image CDN URL</figcaption>
+</figure>
+
+In these situations, if the resource you'll be fetching is important, you want to save as much time as possible by pre-connecting to the server. The browser won't download the file before it needs it (that is, once the request is made from your page somehow), but at least it can handle the connection aspects ahead of time, saving the user from waiting for several round trips.
+
+### Streaming media
+
+Another example where you may want to save some time in the connection phase, but not necessarily start retrieving content right away, is when streaming media from a different origin.
+
+Depending on how your page handles the streamed content, you may want to wait until your scripts have loaded and are ready to process the stream. Pre-connecting helps you cut the waiting time to a single round trip once you're ready to start fetching.
+
+## How to implement `rel=preconnect`
+
+One way of initiating a `preconnect` is adding a `<link>` tag to the `<head>` of the document.
+
+```html
+<head>
+    <link rel="preconnect" href="https://example.com">
+</head>
+```
+
+Pre-connecting is only effective for domains other than the origin domain, so you shouldn't use it for your site. 
+
+{% Aside 'caution' %}
+Only pre-connect to critical origins you will use soon because if the connection isn't used within 10 seconds the browser closes it. There's a limit to the number of simultaneous connections a browser can handle (for example, Chrome can handle six), and unnecessary pre-connecting can delay other important resources.
+{% endAside %}
+
+You can also initiate a preconnect via the [`Link` HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link):
+
+`Link: <https://example.com/>; rel=preconnect`
+
+{% Aside %}
+A benefit of specifying a preconnect hint in the HTTP header is that it doesn't rely on markup being parsed and it can be triggered by requests for stylesheets, scripts and more. For example, Google Fonts sends a `Link` header in the stylesheet response in order to pre-connect the domain that hosts the font files.
```

Comment:
```suggestion
A benefit of specifying a preconnect hint in the HTTP header is that it doesn't rely on markup being parsed, and it can be triggered by requests for stylesheets, scripts, and more. For example, Google Fonts sends a `Link` header in the stylesheet response to pre-connect to the domain that hosts the font files.
```
Comma between clauses, serial comma, idiomatic preposition.